packages/kbn-docs-utils/src/build_api_docs_cli.ts

```diff
@@ -35,9 +38,21 @@ function isStringArray(arr: unknown | string[]): arr is string[] {
   return Array.isArray(arr) && arr.every((p) => typeof p === 'string');
 }
 
+const rootDir = Path.join(__dirname, '../../..');
+initApm(process.argv, rootDir, false, 'build_api_docs_cli');
+
+async function endTransactionWithFailure(transaction: any) {
```

Comment:
You can import the Transaction type:

```
import { Transaction } from 'elastic-apm-node';
```

```suggestion
async function endTransactionWithFailure(transaction: Transaction) {
```